src/js/components/SelectMultiple/SelectMultipleContainer.js

```diff
@@ -0,0 +1,614 @@
+            ref={optionsRef}
+            aria-multiselectable
+            onMouseMove={() => setKeyboardNavigation(false)}
+            aria-activedescendant={optionsRef?.current?.children[activeIndex]}
+          >
+            {options.length > 0 ? (
+              <InfiniteScroll
+                items={options}
+                step={theme.select.step}
+                onMore={onMore}
+                replace={replace}
+                show={activeIndex !== -1 ? activeIndex : undefined}
+              >
+                {(option, index, optionRef) => {
+                  const iValue =
+                    valueKey && valueKey.reduce
+                      ? applyKey(option, valueKey)
+                      : option;
+                  const optionDisabled = checkDisabled(
+                    index,
+                    disabled,
+                    disabledKey,
+                    options,
+                    valueKey || labelKey,
+                  );
+                  const optionSelected = value.includes(iValue);
+                  const optionActive = activeIndex === index;
+                  const iLabel = getOptionLabel(
+                    index,
+                    options,
+                    labelKey || valueKey,
+                  );
+
+                  // Determine whether the label is done as a child or
+                  // as an option Button kind property.
+                  let child;
+                  let textComponent = false;
+                  if (children) {
+                    child = children(option, index, options, {
+                      active: optionActive,
+                      disabled: optionDisabled,
+                      selected: optionSelected,
+                    });
+                    if (
+                      typeof child === 'string' ||
+                      (child.props &&
+                        child.props.children &&
+                        typeof child.props.children === 'string')
+                    )
+                      textComponent = true;
+                  } else {
+                    child = (
+                      <CheckBox
+                        label={
+                          <Box alignSelf="center" width="100%" align="start">
+                            {iLabel}
+                          </Box>
+                        }
+                        pad="xsmall"
+                        tabIndex="-1"
+                        checked={optionSelected}
+                        disabled={optionDisabled}
+                      />
+                    );
+                  }
+
+                  if (!children && !theme.select.options && search) {
+                    if (
+                      typeof iLabel === typeof '' &&
+                      iLabel.toLowerCase().indexOf(search) >= 0
+                    ) {
+                      // code to bold search term in matching options
+                      const boldIndex = iLabel.toLowerCase().indexOf(search);
```

Comment:
How do I test this functionality?

Reply:
When you type in the search you should see substrings that match your search term bolded. I realized that this wasn't working on the grommet theme, but fixed this in the latest commit.